apps/chat/src/utils/app/import-export.ts

```diff
@@ -293,29 +309,29 @@ export const updateAttachment = ({
     return oldRelativePath === cleanOldAttachmentRelativePath;
   });
 
-  if (!newAttachmentFile || !newAttachmentFile.name) {
-    return oldAttachment;
-  }
-
+  const newAttachmentUrl =
+    (oldAttachment.url || oldAttachment.reference_url) &&
+    (newAttachmentFile
+      ? constructPath(newAttachmentFile.absolutePath, newAttachmentFile.name)
+      : constructPath(
+          getFileRootId(BucketService.getBucket()),
```

Comment:
you don't need to provide bucket here
```suggestion
          getFileRootId(),
```
also you can just use `getRootId()`

Reply:
Thank you!
Changed.